docs/Mining/MiningOverview.md

```diff
@@ -0,0 +1,50 @@
+# Mining
+
+Traditional mining is the process of extracting valuable minerals and metals from the earth. These mined minerals are scarce which creates an intrinsic value due to the resources required, the work conducted, and the time it takes to extract. 
+
+## What is Cryptocurrency Mining?
+Cryptocurrency Mining is the process of extracting Cryptocurrencies for securing and verifying transactions made on the blockchain. 
+
+Miners use computers to work on computational puzzles that change their difficulty depending on the number of miners on the network. The difficulty changes to enforce that the blocks are mined at similar time intervals, also known as the Block Generation Time. At each block generated a predetermined amount of cryptocurrency is released to the individual or party who found the solution to the puzzle. This reward is commonly known as the [Block Reward](../Advanced/Cryptoeconomics.md) and is coded to reduce its emission over time.
+
+## Proof-Of-Work
+>> Add information about proof of work.
+
+## Mining Loki
+
+The Loki cryptocurrency can be mined through the CryptoNight Heavy hashing algorithm, which is innately [ASIC Resistant](../Mining/ASICResistance.md). 
+
+### Mining Pools
+
+A mining pool is a hub where a group of miners share their processing power to the network. The rewards are split equally based on the amount of hashing power each individual adds to the block. In the early days of Bitcoin mining it was viable to mine as an individual. However, because there now exists tens of thousands of miners it has become less viable to mine individually due to the share of each individual being minute compared to the overal hash of the network.
```

Comment:
i think this is way to general and not at all related to Loki

Reply:
Mining Pools are related to Loki. When individuals go into the docs they should know that it's better to mine in a pool rather then mine out of the GUI.